firebase-storage/src/main/java/com/google/firebase/storage/network/ResumableUploadByteRequest.java

```diff
@@ -31,15 +30,12 @@
   public ResumableUploadByteRequest(
       @NonNull Uri gsUri,
       @NonNull FirebaseApp app,
-      @NonNull String uploadURL,
+      @NonNull Uri uploadURL,
```

Comment:
The Uri class has a number of verification methods (i.e. isAbsolute(), isHeirarchical(), etc) you could use if you want to verify that it seems reasonable. (I'm not certain what it's *supposed* to look like, so I'm not sure which one(s) would apply). Optional. If you use any of these, also apply to the other Resumable*Request.java files.